blocks/resources/block.php

```diff
@@ -50,6 +51,17 @@ function render_callback( array $attributes ) : string {
 	$resources_query = new AirpressQuery( 'Resources', 0 );
 	$resources_query->addFilter( '{Publish Status of Resource} = "Published"' );
 
+	$trp           = TRP_Translate_Press::get_trp_instance();
+	$url_converter = $trp->get_component( 'url_converter' );
+	$language_code = $url_converter->get_lang_from_url_string();
+
+	if ( $language_code ) {
+		$language_handler = $trp->get_component( 'languages' );
+		$language_name    = $language_handler->get_language_names( array( $language_code ), 'english_name' )[ $language_code ];
+		$resources_query->view( 'Published (' . $language_name . ') Resources grouped by Resource Type - site embed view' );
+	} else {
+		$resources_query->sort( 'Display First', 'desc' );
```

Comment:
Should this be in an `else` statement, or always be called?

Reply:
I am using a view so that each language can have its own sorting order. Deborah and I have been thinking that, for Spanish, we would put Spanish entries first, followed by English. It seemed better to do this in the airtable view so that the team can configure it without code changes.